assessments/guide/security-reviewer.md

```diff
@@ -53,14 +53,37 @@ guidelines for conflict of interest situations are important to prevent:
 - SIG-Security chairs and assessment leads could intentionally or unintentionally limit the participation of an individual unfairly by asserting conflict of interest
 - Security reviews being stalled while groups belabor on who should be allowed to participate
 
-The conflicts of interest lie on a spectrum, and are classified into hard and soft conflicts.
-Hard conflicts makes a reviewer ineligible to review a project.
-A soft conflict allows a reviewer to review a project, but not as a [project lead](./project-lead.md).
-A reviewer with a soft conflict is required to document the nature of the conflict in the
-security assessment documentation. It is the responsibility of the security reviewer
-to make known his/her own conflict of interests.
-
-For each project, 2 SIG-Security chairs must sign off on the conflicts presented to them that the assessment lead has no conflicts, and reviewers have no hard conflicts. Reasons for accepting and rejecting conflicts should be documented.
+The conflicts of interest lie on a spectrum, and are classified into hard and soft conflicts:
+* Hard conflicts make a reviewer ineligible to review a project.
+* A soft conflict allows a reviewer to review a project, but not as a [project lead](./project-lead.md).
+* Soft conflicts are expected for some reviewers, and therefore are fine as long
+  as the review team includes enough different perspectives to ensure a
+  balanced assessment.
+
+A reviewer with a any conflict or potential conflect is required to document the nature of the conflict when volunteering as a reviewer, or at any time as they
+become aware of a potential conflict. It is the responsibility of the security 
+reviewer to make known his/her own conflict of interests and ensure that
+their declaration has been reviewed and approved by lead security reviewer.
+
+The lead security reviewer has the responsibility of ensuring a balanced review,
+and as part of that before kicking off the assessment must:
+* check that all reviewers have conflict-of-interest declarations
+* make their declaration of any potential conflict-of-interest (or lack thereof)
```

Comment:
when reading these bullets, 'their'  and 'they' was a bit ambiguous (referring to reviewers or lead reviewers?) Since there was a mention in bullet 1 with "all reviewers"